packages/ow-db/src/migrations/0011_alter_committee.ts

```diff
@@ -4,7 +4,7 @@ export async function up(db: Kysely<any>) {
   await db.schema
     .alterTable("committee")
     .addColumn("description", "text", (col) => col.notNull().defaultTo(""))
-    .addColumn("email", "text", (col) => col.notNull())
+    .addColumn("email", "text", (col) => col.notNull().defaultTo(""))
```

Comment:
Perhaps make it default to some online.ntnu.no email?

Reply:
Yeah, thats probably a good idea. Maybe `kontakt@online.ntnu.no`?

Reply:
sounds good